tests/integration/common/s2n_test_scenario.py

```diff
@@ -57,6 +57,21 @@ def other(self):
         return Mode.server if self.is_client() else Mode.client
 
 
+class Cert():
+    def __init__(self, name, prefix, location="../pems/"):
+        self.name = name
+        self.cert = location + prefix + "_cert.pem"
+        self.key = location + prefix + "_key.pem"
+
+    def __str__(self):
+        return self.name
+
+ALL_CERTS = [
```

Comment:
/nit can we be more specific and call it ECDSA_CERTS?

Reply:
We should add RSA certs though. We have an issue for that: https://github.com/awslabs/s2n/issues/1613